packages/ui/src/components/shared/menu/ImportAccountDialog/TransferAccountBackup.tsx

```diff
@@ -0,0 +1,123 @@
+import { ChangeEvent, FormEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+import { useToggle } from 'react-use';
+import { AccountBackup } from '@coong/keyring/types';
+import { Alert, Button, TextField } from '@mui/material';
+import AccountCard from 'components/pages/Accounts/AccountCard';
+import useAccountBackupValidation from 'components/shared/menu/ImportAccountDialog/useAccountBackupValidation';
+import EmptySpace from 'components/shared/misc/EmptySpace';
+import useHighlightNewAccount from 'hooks/accounts/useHighlightNewAccount';
+import useAccountNameValidation from 'hooks/useAccountNameValidation';
+import { useWalletState } from 'providers/WalletStateProvider';
+import { Props } from 'types';
+
+interface TransferAccountBackupProps extends Props {
+  backup: AccountBackup;
+  onClose: () => void;
+  resetBackup: () => void;
+}
+
+function TransferAccountBackup({ backup, resetBackup, onClose }: TransferAccountBackupProps): JSX.Element {
+  const { keyring } = useWalletState();
+  const { t } = useTranslation();
+  const [onImporting, setOnImporting] = useToggle(false);
+  const [newName, setNewName] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const { validation } = useAccountNameValidation(newName);
+  const { setNewAccount } = useHighlightNewAccount();
+  const { conflict, resolvable, accountInfo } = useAccountBackupValidation(backup);
+
+  const originalAccountBackupName = backup.meta.name as string;
+
+  if (conflict && !resolvable)
+    return (
+      <div>
+        <AccountCard account={accountInfo} />
+        <Alert severity='error' className='mt-4'>
+          {t<string>(conflict, { name: originalAccountBackupName })}
+        </Alert>
+        <Button onClick={resetBackup} className='mt-4' fullWidth>
+          {t<string>('Back')}
+        </Button>
+      </div>
+    );
+
+  const doImportAccount = (event: FormEvent) => {
+    event.preventDefault();
+
+    setOnImporting(true);
+
+    setTimeout(async () => {
+      try {
+        // To avoid change the content of original backup
+        const cloneBackup: AccountBackup = { ...backup, meta: { ...backup.meta } };
+        cloneBackup.meta.name = newName || originalAccountBackupName;
+
+        const account = await keyring.importAccount(cloneBackup, password);
+        onClose();
+        toast.success(t<string>('Import account successfully'));
+        setNewAccount(account);
+      } catch (e: any) {
+        setOnImporting(false);
+        toast.error(t<string>(e.message));
+      }
+    }, 200);
+  };
+
+  const handleNewNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const newName = e.currentTarget.value;
+    setNewName(newName);
+
+    // To display the preview with the new name
+    // without altering the original backup name
+    if (!newName) accountInfo.name = originalAccountBackupName;
+    else accountInfo.name = newName;
```

Comment:
This is not how to alter a value/state in react, this probably works since the component is rendered because of `setNewName` above, but we should do this properly. 
1. First move the `accountInfo` useMemo outside of the `useAccountBackupValidation`
2. Then trigger recalculate the accountInfo memo when `newName` is changed by adding `newName` to the deps array.